05/DB.test.js

```diff
@@ -123,14 +114,24 @@ describe("class DB", () => {
     it(`REJECTS: the promise | WHEN: provided ID doesn't match any item`, async () => {
       const db = new DB();
 
-      await expect(db.select(1)).rejects.toBeTruthy();
+      await expect(db.select(1)).rejects.toBe("ID not found");
     });
 
     it(`RESOLVES: the promise with selected item as its value | WHEN: provided ID matches any item`, async () => {
+      expect.assertions(2);
+
       const db = new DB();
 
-      await db.insert({ a: 1, b: 2 });
-      await expect(db.select(1)).resolves.toBeTruthy();
+      const newObj = {
+        id: 5,
+        a: 1,
+        b: 2
+      }
+
+      await db.insert(newObj);
+      const insertedObject = await db.select(5);
+      await expect(insertedObject.a).toBe(newObj.a);
+      await expect(insertedObject.b).toBe(newObj.b);
```

Comment:
👍 

PS. Przydatne może być `.toEqual()` (https://jestjs.io/docs/expect#toequalvalue) - nie porównuje czy to ten sam obiekt, ale czy ta sama zawartość.